aws_cloudtrail_rules/aws_iam_entity_created_without_cloudformation.py

```diff
@@ -26,5 +33,10 @@ def rule(event):
         return True
 
     # Only approved IAM Roles can make IAM Changes
-    return event['userIdentity']['sessionContext']['sessionIssuer'][
-        'arn'] not in IAM_ADMIN_ROLES
+    for admin_role_pattern in IAM_ADMIN_ROLES:
```

Comment:
Should this be `for admin_role_pattern in IAM_ADMIN_ROLE_PATTERNS:` ?

Reply:
Doh, thanks for that catch